packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2134,7 +2133,8 @@ async function handler(
       }
       return prev;
     }, {} as { [key: string]: AppsStatus });
-    evt.appsStatus = Object.values(calcAppsStatus);
+    resultStatus = Object.values(calcAppsStatus);
+    return resultStatus;
```

Comment:
Return `resultStatus` instead of setting it directly on evt in the function. evt isn't an argument passed to the function.